tests/dummy/app/pods/range-picker/controller.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember'
+
+export default Ember.Controller.extend({
+  actions: {
+    log (e) {
+      // perform validation here
+      console.log(e)
```

Comment:
notifications would be nice here. also, example of validation and put it in a snippet